src/App.js

```diff
@@ -6,30 +6,39 @@ import ChatBotComponent from './components/chatbot';
 import './App.css';
 import Board from './board.jpg';
 import Footer from './components/Footer';
+import { ThemeProvider } from 'styled-components';
+import { useDarkMode } from './components/useDarkMode';
+import Toggle from './components/Toggle';
+import { lightTheme, darkTheme } from './components/themes';
```

Comment:
it should be ./component/theme

Reply:
@sagar-barapatre I am not able to extract I have no file or folder named theme Its just themes.js Also the folder is named component right?

Reply:
@sagar-barapatre  I have changed file locations kindly see if this worked